src/models/curriculum/problem.ts

```diff
@@ -116,22 +117,29 @@ const isAmbiguousSnapshot = (sn: ModernProblemSnapshot | LegacyProblemSnapshot)
 
 export const ProblemModel = types.snapshotProcessor(ModernProblemModel, {
   preProcessor(sn: ModernProblemSnapshot | LegacyProblemSnapshot) {
-    const { sections, ...nonSectionProps } = sn as any;
+    const { sections, exemplars, ...nonSectionProps } = sn as any;
     // Move sections to sectionsFromSnapshot so we load them into a volatile property `sections`
+    // creating a exemplarsFromSnapshot that will be passed in similar way,
+    // but no volatile property yet
```

Comment:
This comment is out of date.

I think you can revert all of the changes to `snapshotProcessor` in this PR. The exemplars will get passed through via `others` in most cases and in the default case they will get passed through via `nonSectionProps` if they aren't deconstructed.